contracts/CDPosition.sol

```diff
@@ -53,6 +53,10 @@ contract CDPosition {
         external
         nftIDMustExist(nftID)
     {
+        // sainty check
```

Comment:
Nitpick -better say what and why it checks this rather then a general sanity check comment